resources/web/docs_js/actions/settings.js

```diff
@@ -0,0 +1,39 @@
+import Cookies from "../../../../../node_modules/js-cookie";
```

Comment:
Why are you importing from `node_modules` explicitly instead of doing `import Cookies from 'js-cookie';` and letting Parcel's resolver find it for you?

Reply:
Yea, kind of stupid impl detail we're working with at the moment. We install js dependencies in the 'host' docker image, so node_modules are up a directory from where they are when you install it locally. Parcel doesn't have an easy way to say "here's the path to node_modules" (ironically CLJS does :P), and I couldn't get it working with native package.json solutions because Parcel implements a custom resolver. So, for now, we resort to relative paths that target the folder.

Reply:
FWIW [Webpack also has a resolver API that's pretty well documented](https://webpack.js.org/concepts/module-resolution/) that solves for exactly that. Not sure if there's a counterpart in Parcel that's easy to configure.

Reply:
Unfortunately not and I thought about switching the entire thing to Webpack more than once, or rollup. Ultimately decided to deal with the relative path for now.